server/app/views/NorthStarBaseView.java

```diff
@@ -97,11 +97,13 @@ protected ThymeleafModule.PlayThymeleafContext createThymeleafContext(
     // that will be embedded in the guest alert in the header.
     context.setVariable(
         "endSessionLinkHtml",
-        "<a id=\"logout-button\" class=\"usa-link\" href=\""
-            + logoutLink
-            + "\">"
-            + messages.at(MessageKey.END_YOUR_SESSION.getKeyName())
-            + "</a>");
+        profile.isPresent()
```

Comment:
I am confused - it looks like for a user with no profile this would show "End session" it just wouldn't be linked? Shouldn't it show nothing at all in that case?

I tried to run your PR locally to confirm that's what would happen, but it's showing the link even when I delete the session so I'm confused about that too...

Reply:
What would the new message be? I was thinking it wouldn't say anything at all which wouldn't require any translations.